python-mini-projects/projects/Fetch_and_store_tweets/fetch_store_tweet.py

```diff
@@ -0,0 +1,24 @@
+import tweepy
+import csv
+
+# input your credentials here
+consumer_key = ''
+consumer_secret = ''
+access_token = ''
+access_token_secret = ''
+hastag = ''
+
+auth = tweepy.OAuthHandler(consumer_key, consumer_secret)
+auth.set_access_token(access_token, access_token_secret)
+api = tweepy.API(auth,wait_on_rate_limit=True)
+
+# Open/Create a file to append data
+csvFile = open('tweets.csv', 'a')
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *csvFile*. The resource is allocated by call *builtins.open*. Execution paths that do not contain closure statements were detected. To prevent this resource leak, close *csvFile* in a try-finally block or declare it using a `with` statement.

**More info**
[View details about the `with` statement in the Python developer&#39;s guide](https://www.python.org/dev/peps/pep-0343/) (external link).